### PR TITLE
Remove non-prepared variant of database select wrappers

### PR DESCRIPTION
**Pull request by driusan**
The non-prepared variant of the database select wrappers have been
deprecated for a long time, but left around because of the need of
projects to update any instruments they might have that uses them.

This finally removes the code, now that users have had time to
update any calls to them and all references in the LORIS codebase
have been removed.

See also: comments on #3544

### CODE REVIEW

**Comment by johnsaigle** [comment]
This is a very good idea Dave! Happy that it's happening.

**Comment by ridz1208** [comment]
@driusan 35 seconds later... you have a conflict

File: php/libraries/Database.class.inc [hunk 1]
```diff
@@ -549,106 +549,6 @@ class Database
...
-     *
-     * @param string $query  the SQL string which defines the SELECT query.
-     * @param array  $result an array into which the results of the select will
-     *                       be stored - this array gets overwritten completely
-     *
-     * @return     void
-     * @access     public
-     * @deprecated Use pselect wrappers instead
-     */
-    function selectRow($query, &$result)
```
Comment (xlecours) [inline php/libraries/Database.class.inc]: `docs/instruments/NDB_BVL_Instrument_mri_parameter_form.class.inc` will be sad

File: php/libraries/Database.class.inc [hunk 2]
```diff
@@ -549,106 +549,6 @@ class Database
...
-     *
-     * @param string $query  the SQL string which defines the SELECT query.
-     * @param array  $result an array into which the results of the select
-     *                       will be stored - this array gets overwritten completely
-     *
-     * @return     none
-     * @access     public
-     * @deprecated Use pselect wrappers instead
-     */
-    function select($query, &$result)
```
Comment (xlecours) [inline php/libraries/Database.class.inc]: poor `tools/excelDump.php`

**Comment by ridz1208** [comment]
@xlecours Done and Done please re-review

File: php/libraries/Database.class.inc (hunk 1 above)
Comment (ridz1208) [reply php/libraries/Database.class.inc]: resolved

File: php/libraries/Database.class.inc (hunk 2 above)
Comment (ridz1208) [reply php/libraries/Database.class.inc]: resolved

**Pull request merged**